.github/workflows/php.yml

```diff
@@ -0,0 +1,23 @@
+name: PHP Composer
```

Comment:
Please remove this file: I don't think this should be added to this commit. If you think this should be added, please open another PR explaining why.